### DISCUSSION TITLE
Terraform Lockfile limitation

### DISCUSSION THREAD

**Discussion by renan-alm** [renovatebot/renovate#11740] [Request Help] [answered]
Hi,

I'm currently using a self-hosted instance of Renovate to monitor Terraform and I have noticed that when Renovate updates the Terraform lockfile, only the first provider gets updated (all the others remain untouched). Am I missing in configuration here or is it a known issue/feature/limitation? 

Kind regards,
Renan A.

**Comment by viceice** [comment, marked as answer]
fixed in [27.6.0](https://github.com/renovatebot/renovate/releases/tag/27.6.0) also live in hosted app
